From f078fa9dcbe6c1f9d4487d51527590f9153cd194 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Wed, 9 Nov 2011 13:36:17 +0800
Subject: [PATCH 0583/2463] ENGR00161755 MX6Q/MFGTOOL : disable GPU for
 MFGTOOL

The GPU module will eat up lots of memory.
So disable GPU to save same memory.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/configs/imx6_updater_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6_updater_defconfig b/arch/arm/configs/imx6_updater_defconfig
index b44c4fd..f9ab946 100644
--- a/arch/arm/configs/imx6_updater_defconfig
+++ b/arch/arm/configs/imx6_updater_defconfig
@@ -1953,7 +1953,7 @@ CONFIG_MXC_ASRC=y
 #
 # MXC GPU support
 #
-CONFIG_MXC_GPU_VIV=y
+# CONFIG_MXC_GPU_VIV is not set
 
 #
 # ANATOP_THERMAL
-- 
1.7.7.4

